google/auth/credentials.py

```diff
@@ -128,6 +128,10 @@ def universe_domain(self):
         """The universe domain value."""
         return self._universe_domain
 
+    def get_cred_info(self):
+        """The credential information JSON."""
```

Comment:
nit: can this comment describe the JSON format and what it gets used for

Reply:
done